### PR TITLE
Can restrict conveyor belt switches to one direction with multitool

### PR DESCRIPTION
**Pull request by 9600bauds**
Because an anon in the thread asked for it

### CODE REVIEW

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -425,24 +434,26 @@
...
+			<li><b>Restrict Pulling: </b>
+				[src.oneway && src.convdir == -1 ? "<b>&larr;</b>" : {"<a href="?src=\ref[src];toggleoneway=-1">&larr;</a>"}]
+				[!src.oneway ? "<b>No</b>" : {"<a href="?src=\ref[src];toggleoneway=0">No</a>"}]
+				[src.oneway && src.convdir == 1 ? "<b>&rarr;</b>" : {"<a href="?src=\ref[src];toggleoneway=1">&rarr;</a>"}]
+			</li>
+		</ul>"}
+
+/obj/machinery/conveyor_switch/multitool_topic(var/mob/user,var/list/href_list,var/obj/O)
+	. = ..()
+	if(.) return .
```
Comment (PJB3005) [inline code/modules/recycling/conveyor2.dm]: `return .`

AAAAAAahaaahaAAAAAAAAAAAAhahahahaa

Comment (9600bauds) [inline code/modules/recycling/conveyor2.dm]: :^)

It was already like that in the conveyor belt multitool menu
I thought it was funny

Comment (PJB3005) [inline code/modules/recycling/conveyor2.dm]: AAAAAAAAAAAAAahahahaaaaaaaaaaaaaahahahaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by 9600bauds** [comment]
In hindsight you're right.

**Pull request merged**